src/lib/markbind/src/parser.js

```diff
@@ -166,12 +166,39 @@ Parser.prototype.getMissingIncludeSrc = function () {
   return _.clone(this.missingIncludeSrc);
 };
 
+Parser.prototype._preprocessThumbnails = function (node) {
+  const element = node;
```

Comment:
Since the function call from `_preprocess` already name the argument as `element`, why does your method change the parameter name to `node` then put it in an `element` const?

We can omit this line if the parameter name is `element` from the start.

Reply:
This is done to suppress eslint errors regarding parameter reassignment (`no-param-reassign`), similar to what is done in `_preprocess`

The alternative is to use `// eslint-disable-next-line no-param-reassign`. In this case this is fine since there is only one occurrence, but I decided to follow the existing "fix"